Sources/KSCrashInstallations/KSCrashInstallation+Private.h

```diff
@@ -24,66 +24,63 @@
 // THE SOFTWARE.
 //
 
-
 #import "KSCrashInstallation.h"
 
-
 /** Implement a property to be used as a "key". */
-#define IMPLEMENT_REPORT_KEY_PROPERTY(NAME, NAMEUPPER) \
-@synthesize NAME##Key = _##NAME##Key; \
-- (void) set##NAMEUPPER##Key:(NSString*) value \
-{ \
-    _##NAME##Key; \
-    _##NAME##Key = value; \
-    [self reportFieldForProperty:@#NAME setKey:value]; \
-}
+#define IMPLEMENT_REPORT_KEY_PROPERTY(NAME, NAMEUPPER)      \
+    @synthesize NAME##Key = _##NAME##Key;                   \
+    -(void)set##NAMEUPPER##Key : (NSString*)value           \
+    {                                                       \
+        _##NAME##Key;                                       \
+        _##NAME##Key = value;                               \
+        [self reportFieldForProperty:@ #NAME setKey:value]; \
+    }
 
 /** Implement a property to be used as a "value". */
 #define IMPLEMENT_REPORT_VALUE_PROPERTY(NAME, NAMEUPPER, TYPE) \
-@synthesize NAME = _##NAME; \
-- (void) set##NAMEUPPER:(TYPE) value \
-{ \
-    _##NAME; \
-    _##NAME = value; \
-    [self reportFieldForProperty:@#NAME setValue:value]; \
-}
+    @synthesize NAME = _##NAME;                                \
+    -(void)set##NAMEUPPER : (TYPE)value                        \
+    {                                                          \
+        _##NAME;                                               \
+        _##NAME = value;                                       \
+        [self reportFieldForProperty:@ #NAME setValue:value];  \
+    }
```

Comment:
This looks dangerous, but kinda fine. Always nervous to see macros formatting

Reply:
This is best I could get with `AlignEscapedNewlines: Left`

Reply:
It's fine, I even like it, let's keep it